install/index.php

```diff
@@ -18,8 +18,8 @@
  */
 
 // The new version of THT we're installing
-define("NVER", "1.3");
-define("NVERCODE", 1010300);
+define("NVER", "1.4.0");
+define("NVERCODE", 1010400);
```

Comment:
The rss classes had to be removed, and a lot of the code had to be updated to be compatible with mysqli, I can change it back to 1.3.0 if you want.